oauthproxy.go

```diff
@@ -28,7 +28,8 @@ import (
 	"github.com/oauth2-proxy/oauth2-proxy/pkg/middleware"
 	"github.com/oauth2-proxy/oauth2-proxy/pkg/sessions"
 	"github.com/oauth2-proxy/oauth2-proxy/pkg/upstream"
-	"github.com/oauth2-proxy/oauth2-proxy/providers"
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/validation"
+  "github.com/oauth2-proxy/oauth2-proxy/providers"
```

Comment:
Please fix this indentation

Reply:
Not sure why you are seeing indentation errors. When I'm grepping the code, I don't see it:
[kfox@zathras oauth2_proxy]$ grep '^ ' oauthproxy.go
[kfox@zathras oauth2_proxy]$